src/common/test-json.c

```diff
@@ -117,6 +117,31 @@ test_get_int (TestCase *tc,
 }
 
 static void
+test_get_double (TestCase *tc,
+                 gconstpointer data)
+{
+  gboolean ret;
+  gdouble value;
+
+  ret = cockpit_json_get_double (tc->root, "number", 0, &value);
+  g_assert (ret == TRUE);
+  g_assert_cmpint (value, ==, 55.4);
```

Comment:
Erps, sorry. I thought I used ```_cmpfloat()``` but forgot to push. I guess we'll see in practice whether we need more that ```_cmpfloat()```